src/components/PeoplePage/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Person } from '../../../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = props => {
+  const { people } = props;
+
+  const peopleWithParents = people.map(person => {
+    const mother = people.find(m => person.motherName === m.name);
+    const father = people.find(f => person.fatherName === f.name);
+
+    return {
+      ...person,
+      mother,
+      father,
+    };
+  });
```

Comment:
Do it in the parent(container) component. `PeopleTable` is a presentational component in your case so avoid keeping any logic in it
https://www.freecodecamp.org/news/separation-of-concerns-react-container-and-presentational-components/